.github/workflows/python.yml

```diff
@@ -197,101 +165,36 @@ jobs:
         instance: [0]
 
     env:
-      SENTRY_PYTHON3: 1
-      PIP_DISABLE_PIP_VERSION_CHECK: on
-
+      # Note: `USE_SNUBA` is only used for the Snuba test suite because we have some failing acceptance tests with Snuba enabled.
       USE_SNUBA: 1
-      SENTRY_LIGHT_BUILD: 1
-      SENTRY_SKIP_BACKEND_VALIDATION: 1
       MIGRATIONS_TEST_MIGRATE: 1
-
-      PYTEST_SENTRY_DSN: https://6fd5cfea2d4d46b182ad214ac7810508@sentry.io/2423079
-      PYTEST_ADDOPTS: "--reruns 3"
-      PYTEST_SENTRY_ALWAYS_REPORT: yes
-
-      # services configuration
-      SENTRY_REDIS_HOST: redis
-      DATABASE_URL: postgresql://postgres:postgres@localhost/sentry
-      SENTRY_KAFKA_HOSTS: localhost:9092
-      SENTRY_ZOOKEEPER_HOSTS: localhost:2181
-
-      # Number of matrix instances
-      TOTAL_TEST_GROUPS: ${{ strategy.job-total }}
+      TOTAL_TEST_GROUPS: ${{ strategy.job-total }} # Number of matrix instances
 
     steps:
-      - name: Install System Dependencies
-        run: |
-          sudo apt-get update
-          sudo apt-get install -y --no-install-recommends \
-            libxmlsec1-dev \
-            libmaxminddb-dev
-
       - uses: actions/checkout@v2
 
-      - name: Set up outputs
-        id: config
-        env:
-          MATRIX_INSTANCE: ${{ matrix.instance }}
-        run: |
-          echo "::set-output name=python-version::$(awk 'FNR == 2' .python-version)"
-          echo "::set-output name=matrix-instance-number::$(($MATRIX_INSTANCE+1))"
+      - uses: volta-cli/action@v1
 
-      - name: Set up Python ${{ steps.config.outputs.python-version }}
-        uses: actions/setup-python@v2
+      # Until GH composite actions can use `uses`, we need to setup python here
+      - uses: actions/setup-python@v2
         with:
-          python-version: ${{ steps.config.outputs.python-version}}
+          python-version: 2.7.17
 
-      - name: Install pip
-        run: |
-          pip install --no-cache-dir --upgrade "pip>=20.0.2"
-
-      - name: Get pip cache dir
-        id: pip-cache
-        run: |
-          echo "::set-output name=dir::$(pip cache dir)"
-
-      - name: pip cache
-        uses: actions/cache@v1
+      - name: Setup sentry env
+        uses: ./.github/actions/setup-sentry
+        id: setup
         with:
-          path: ${{ steps.pip-cache.outputs.dir }}
-          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements-*.txt') }}
-          restore-keys: |
-            ${{ runner.os }}-pip-
+          python: 2
+          kafka: 1
 
-      - name: Install Python Dependencies
-        env:
-          PGPASSWORD: postgres
+      - name: Install rb
```

Comment:
You forgot to remove this rb.

Reply:
Oh, you just haven't gotten around to that yet. I see the other one too.

Reply:
oop